src/simulator/model.ts

```diff
@@ -20,3 +20,17 @@ export interface EnumSensor {
   choices: string[];
   value: string;
 }
+
+export interface LogData {
+  key: string;
+  value: string | number;
+}
+
+export interface DataLoggingSensor {
+  type: "log";
+  id: string;
+  value: [LogData[]];
```

Comment:
Do we mean a singleton array here?

Reply:
It's currently not, due to some difficulties I was having, but it probably should be.